### PR TITLE
Show kick reason

### PR DESCRIPTION
**Pull request by Brentdevent**
Ported the kick/drop reason from S1x to IW6x.

### CODE REVIEW

**Comment by efinst0rm** [comment]
@Brentdevent Maybe it would be a good idea keeping clientkick as is since it would be constant with the other cods.

**Comment by Brentdevent** [comment]
> @Brentdevent Maybe it would be a good idea keeping clientkick as is since it would be constant with the other cods.

I copied it over from S1x but I see `clientkick` Is also used in IW4x. So I guess you're right. I'll revert it and probably also change it for S1x.

File: src/client/component/party.cpp [hunk 1]
```diff
@@ -125,6 +125,26 @@ namespace party
 
 			return count;
 		}
+
+		void disconnect_stub()
+		{
+			if (game::CL_IsCgameInitialized())
+			{
+				// CL_ForwardCommandToServer
+				reinterpret_cast<void (*)(int, const char*)>(0x1402C76C0)(0, "disconnect");
```
Comment (diamante0018) [inline src/client/component/party.cpp:134]: Wouldn't it be better to replace these reinterpret casts with the appropriate game 'symbol'. (Add functions to symbol file)

Comment (momo5502) [reply src/client/component/party.cpp:134]: Fuck that 😂

**Review by momo5502 — APPROVED** [review]

**Pull request merged**